packages/SwingSet/test/vat-admin/terminate/bootstrap-die-cleanly.js

```diff
@@ -15,7 +15,7 @@ export function buildRootObject() {
     },
     async phase2() {
       // terminate as a second phase, so we can capture the kernel state in between
-      E(dude.adminNode).terminate('phase 2');
+      E(dude.adminNode).terminateWithFailure('phase 2');
```

Comment:
Sure we should have at least some tests with a non-error reason, such as a string, to be sure it works. But when that's not the point of the test, we should normally use an error as the reason.